src/server/routes/configuration/db/list.ts

```diff
@@ -0,0 +1,40 @@
+import { Type, type Static } from "@sinclair/typebox";
+import type { FastifyInstance } from "fastify";
+import { StatusCodes } from "http-status-codes";
+import { ConfigurationDB } from "../../../../db/configuration/db";
+import {
+  configurationSchema,
+  toConfigurationSchema,
+} from "../../../schemas/configuration";
+import { standardResponseSchema } from "../../../schemas/sharedApiSchemas";
+
+const responseBodySchema = Type.Object({
+  result: configurationSchema,
+});
+
+export async function listConfiguration(fastify: FastifyInstance) {
+  fastify.route<{
+    Reply: Static<typeof responseBodySchema>;
+  }>({
+    method: "GET",
+    url: "/configuration",
+    schema: {
+      summary: "List configuration",
+      description:
+        "List configuration values. Note: This does not include legacy configuration.",
```

Comment:
i see this is the new way to add / fetch config flags?

seems a bit weird to me to use redis for it though no?